index.html

```diff
@@ -15,12 +15,6 @@
     <footer is="c-footer"></footer>
   </div>
 
-  <script>
-    const payload = [{ "message": "This notification was delivered trough initial data.", "type": "success" }];
-
-    window.initialData['notification/data'] = payload;
```

Comment:
Do we keep the logic to parse initial notificatations from the backend in the file `store/index.js` now that you have removed that example here? If yes, please also adjust the wording that it uses notifications instead of messages there

Reply:
I just removed the example from the index.html file